oic.wk.col.atomic.raml

```diff
@@ -0,0 +1,238 @@
+#%RAML 0.8
+#RAML for Atomic Measurement.
+#
+title: Atomic Measurement
+version: 0.1
+documentation:
+
+  - title: OCF Copyright
+    content: Copyright (c) 2016, 2017 Open Connectivity Foundation, Inc. All rights reserved.
+  - title: OCF License
+    content: !include LICENSE.md
+  - title: OCF Disclaimer
+    content: !include DISCLAIMER.md
+  - title: Atomic Measurement
+    content: |
+      Atomic Measurement is a specialization of the OCF Collection, which is an
+      extension of OIC Resource. In addition to properties, collections also
+      have an array of OIC web links which point to the items in the collection.
+
+      Atomic Measurement applies a special restricton in that the items
+      in the collection may only be accessed through the collection interfaces,
+      oic.if.ll and oic.if.b, and that the items pointed to in the Atomic
+      Measurement must be hosted locally, that is on the same platform as the
+      Atomic Measurement resource itself.
+
+  - title: Atomic Measurement Restrictions on included resources
+    content: |
+      Atomic Measurement restricts properties to be exposed only through the
+      oic.if.b interface of the Atomic Measurement, thus insuring that all
+      of the items are accessed "atomically" as a batch.
+
+      If a client attempts to access an item in an Atomic Measurement individually,
```

Comment:
Remove descriptive text that is a duplicate of the Core spec.  Also do not put any "normative" text in comments.